core_lib/src/util/pencilerror.h

```diff
@@ -71,6 +71,11 @@ class Status
 
         // Layer
         ERROR_NEED_AT_LEAST_ONE_CAMERA_LAYER,
+
+        // Timeline
+        KEYFRAME_MODIFIED,
+        KEYFRAME_MOVED,
+        KEYFRAME_EXISTS
```

Comment:
Seems these 3 values are not really used? In `saveKeyFrameFile()` it only checks whether it's `Status::OK`.
And Status is designed for error report, so if you like to return some specific values other than a true/false, probably making a plain enum is a better idea.

Reply:
👍  forgot to remove from an earlier test